### DISCUSSION TITLE
how add text without treat/style it it as `.md` format?

### DISCUSSION THREAD

**Discussion by xmaxrayx** [quarto-dev/quarto-cli#10350] [Q&A] [answered]
quarto html

### Description

````
---
title: "License"
---

{{< include Assists/gpl3.txt >}}

````

![image](https://github.com/user-attachments/assets/4a475ac0-15f1-4961-9283-f2e3c592c1c2)

**Comment by mcanouil** [comment]
Why do you have backtick and single quote to encapsulate content?
That is not markdown syntax.
Backticks means the content is code.

See <https://quarto.org/docs/authoring/markdown-basics.html>.

If you want actual backtick, you need to escape it.

As documented the include shortcode is a copy/paste of the content inside the main document.

Please note that your example is not reproducible, which means we are unable to use it and therefore find it challenging to assist you effectively.
Additionally, setting a title does not mean your post can consist solely of a code block without any context or accompanying text. Kindly ensure your message is intelligible by adding relevant text.

**Reply by xmaxrayx** [reply]
> Why do you have backtick and single quote to encapsulate content? That is not markdown syntax. Backticks means the content is code.
> 
> See https://quarto.org/docs/authoring/markdown-basics.html.
> 
> If you want actual backtick, you need to escape it.
> 
> As documented the include shortcode is a copy/paste of the content inside the main document.
> 
> Please note that your example is not reproducible, which means we are unable to use it and therefore find it challenging to assist you effectively. Additionally, setting a title does not mean your post can consist solely of a code block without any context or accompanying text. Kindly ensure your message is intelligible by adding relevant text.


yes its text file, i just want add it without being treated as md, i don't want re-edit the file because it wont make it original and more work.

this the file
[agpl3.txt](https://github.com/user-attachments/files/16331621/agpl3.txt)

**Comment by xmaxrayx** [comment]
this work but it removes new line and link so it mades the code text unreadable
```
\````{=s5}
{{< include gpl3.txt >}}

\````
```
![image](https://github.com/user-attachments/assets/653a67d4-ad76-46a3-be93-8db7bd465a1b)


text and plain format ,wont do anything you just get empty result.

**Comment by xmaxrayx** [comment]
this is a work around

\````

{{< include gpl3.txt >}}


\````

**Comment by xmaxrayx** [comment, marked as answer]
just do this 

\~~~text
{{< include gpl3.txt >}}

\~~~



\```text

{{< include gpl3.txt >}}

\```
